FreeRTOS/FreeRTOS
# Fix warnings and update ReadMe

Description
-----------
The demo `FreeRTOS-Plus/Demo/FreeRTOS_Plus_TCP_IPv6_Demo/IPv6_Multi_WinSim_demo` builds clean without warning. Also, make the demo output comprehensible.

Test Steps
-----------
Tested locally.

Checklist:
----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
